src/main/proto/com/google/api/codegen/samplegen/v1/sample_config_v1.proto

```diff
@@ -9,10 +9,10 @@ option java_package = "com.google.api.codegen.samplegen.v1";
 message SampleConfigProto {
   string schema_version = 1;  // current version is "1.2"
 
-  repeated SampleSpec samples = 2;
+  repeated SampleSpecProto samples = 2;
```

Comment:
Renamed a bunch of top-level messages to `XxxProto` to be consistent with gapic_config.yaml. This also makes generation code more readable.

Reply:
Since we're getting rid of `gapic_config.yaml`, I don't see consistency with that as being important.

Readability is a more important concern. I had suggested not having the `Proto` suffix because it's redundant in the `.proto` file (all messages are protocol buffers there), but I can see that if the name transfers over to the generated code it would make for more readable code on this side.....